modules/oadp-gcp-wif-cloud-authentication.adoc

```diff
@@ -96,3 +95,8 @@ $ oc create namespace <OPERATOR_INSTALL_NS>
 ----
 $ oc apply -f manifests/openshift-adp-cloud-credentials-gcp-credentials.yaml
 ----
+
+[id="oadp-gcp-wif-known-issues"]
+== Google workload identity federation known issues
+
+* Volume Snapshot Locations (VSL) backups finish with a `PartiallyFailed` phase when GCP workload identity federation is configured. Google workload identity federation authentication does not support VSL backups.
```

Comment:
Should this be singular? I see 3 references to it in our docs and it seems more common in web searches (plus _Locations backups_ sounds weird).
 ```suggestion
* Volume Snapshot Location (VSL) backups finish with a `PartiallyFailed` phase when GCP workload identity federation is configured. Google workload identity federation authentication does not support VSL backups.
```